recipes/outgoingmessage-headers/codemod.yaml

```
@@ -0,0 +1,22 @@
+schema_version: "1.0"
+name: "@nodejs/outgoingmessage-headers"
```

```suggestion
name: "@nodejs/http-outgoingmessage-headers"
```

not fan of the name but my proposition did look right @nodejs/userland-migrations any idea ?